airflow/hooks/druid_hook.py

```diff
@@ -83,6 +84,9 @@ def construct_ingest_query(
         else:
             num_shards = -1
 
+        query_granularity = query_granularity or "NONE"
+        segment_granularity = segment_granularity or "DAY"
```

Comment:
I meant it from the Python side. In the new version of the hook, you add two new required arguments to the `contruct_ingest_query` and then providing defaults inside the function, which is not backwards compatible (people calling it without these will get an error).

You could just add those defaults to the function definition as default values for keyword arguments
`query_granularity="NONE", segment_granularity="DAY"` in the function definition, which will allow people to use the same interface that was defined before and will ensure the same result you want. I wonder where the best place to put those defaults is, whether they should also exist in send_ingest_query.